### ISSUE TITLE
Equalizer - No option for specific media queries

### ISSUE THREAD

**Issue by Ne-Ne** [zurb/foundation-sites#8864]
Hi All,

Looking for a foundation solution, can we include support for 'only' breakpoints?

E.g.

```html
<div class="row" data-equalizer data-equalize-on="medium" id="test-eq">
...
</div>
```

This solution with give users the power to be specific range, in my case the page design needs to to be equal height for medium only. Large and small are completely different. The only solution is to use `!important` as an override which terrible to have to use..

**Comment by Ne-Ne** [comment]
Any ideas @gakimball @zurbchris?

**Comment by kball** [comment]
Addressed in #9008, will be in 6.3
